src/Persistence/Contexts/ApplicationDbContext.cs

```diff
@@ -22,5 +22,16 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : ba
         public DbSet<PublicHoliday> PublicHolidays { get; set; }
         public DbSet<Team> Teams { get; set; }
 
+
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+
+            builder.Entity<Team>().HasData(
+                new Team() { Id = 1, Name = "Team1" },
+                new Team() { Id = 2, Name = "Team2" }
+                );
+        }

```

Comment:
Create issue to fix this before pushing to prod since this isn't how we do seeding

Reply:
Done